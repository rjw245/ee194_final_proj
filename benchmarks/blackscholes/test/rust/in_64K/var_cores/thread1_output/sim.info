{'cmdline': ['/h/rwood04/ee194/sniper/run-sniper',
             '-d',
             './thread1_output',
             '-n',
             '1',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../../rust/bs',
             '1',
             '../../../inputs/in_64K.txt'],
 'git_revision': '',
 'host': 'dell26',
 'pin_version': 71313,
 'rusage': (12655.781028,
            5.946096,
            109532,
            0,
            0,
            0,
            375912,
            0,
            0,
            0,
            11200,
            0,
            0,
            0,
            252315,
            775409),
 'snipercmd': ['bash',
               '-c',
               '/h/rwood04/ee194/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/rwood04/ee194/sniper/lib/pin_sim -c /h/rwood04/ee194/sniper/config/base.cfg --general/total_cores=1 --general/output_dir=/h/rwood04/ee194/ee194_docs/final_project/ee194_final_proj/benchmarks/blackscholes/test/rust/in_64K/thread1_output --config=/h/rwood04/ee194/sniper/config/nehalem.cfg --config=/h/rwood04/ee194/sniper/config/gainestown.cfg --config=/h/rwood04/ee194/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../../rust/bs 1 ../../../inputs/in_64K.txt'],
 't_elapsed': 17773.523884058,
 't_start': 1460954529.457939,
 'tracecmds': [],
 'user': 'rwood04',
 'vmem': 3196952576L}
