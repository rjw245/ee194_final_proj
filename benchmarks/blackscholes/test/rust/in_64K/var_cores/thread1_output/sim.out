                                     | Core 0     
  Instructions                       |  4326689610
  Cycles                             |  2564586864
  IPC                                |        1.69
  Time (ns)                          |   964130400
  Idle time (ns)                     |        3820
  Idle time (%)                      |        0.0%
Branch predictor stats               |            
  num correct                        |   151241976
  num incorrect                      |     6935578
  misprediction rate                 |       4.38%
  mpki                               |        1.60
TLB Summary                          |            
  I-TLB                              |            
    num accesses                     |   875448320
    num misses                       |         195
    miss rate                        |       0.00%
    mpki                             |        0.00
  D-TLB                              |            
    num accesses                     |  2875368158
    num misses                       |      162875
    miss rate                        |       0.01%
    mpki                             |        0.04
  L2 TLB                             |            
    num accesses                     |      163070
    num misses                       |        1077
    miss rate                        |       0.66%
    mpki                             |        0.00
Cache Summary                        |            
  Cache L1-I                         |            
    num cache accesses               |   875448320
    num cache misses                 |    20495576
    miss rate                        |       2.34%
    mpki                             |        4.74
  Cache L1-D                         |            
    num cache accesses               |  2875368658
    num cache misses                 |      131761
    miss rate                        |       0.00%
    mpki                             |        0.03
  Cache L2                           |            
    num cache accesses               |    20627449
    num cache misses                 |       65834
    miss rate                        |       0.32%
    mpki                             |        0.02
  Cache L3                           |            
    num cache accesses               |       65954
    num cache misses                 |       33698
    miss rate                        |      51.09%
    mpki                             |        0.01
DRAM summary                         |            
  num dram accesses                  |       33693
  average dram access latency (ns)   |       58.88
  average dram queueing delay        |        5.45
  average dram bandwidth utilization |       0.00%
Coherency Traffic                    |            
  num loads from dram                |        4147
  num loads from dram cache          |           0
  num loads from remote cache        |           0
